appium-tests/common/common.spec.js

```diff
@@ -242,7 +248,6 @@ describe('Contacts Android', function () {
 
     describe('Picking contacts', function () {
         afterEach(function (done) {
-            checkSession(done);
```

Comment:
OK, I will add them back if you feel like they are beneficial. I trust you as you'd have much more exposure to the CI than I have :D